commands/import.go

```diff
@@ -224,6 +260,46 @@ func (i *Importer) run() error {
 	return nil
 }
 
+func (i *Importer) handleDeleteOldData() error {
+	if !i.res.Config.S.Rolling.Rolling {
+		fmt.Printf("\t[+] Removing database: %s\n", i.targetDatabase)
```

Comment:
Added some print statements and some error handling to pretty this section up a bit.

In particular the call to `removerRepo.Remove` prints to the screen, so I wanted to make sure the something was printed for non rolling databases as well.